mgit/lib/commands/exec.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const utils = require( '../utils' );
+
+module.exports = {
+	/**
+	 * @param {Object} data
+	 * @param {Object} data.parameters Additional arguments provided by user.
+	 * @param {String} data.name Name of current package that will be parsed.
+	 * @param {Object} data.mgit MGit configuration.
+	 * @param {Object} data.mgit.dependencies MGit dependencies.
+	 * @returns {Promise}
+	 */
+	execute( data ) {
+		return new Promise( ( resolve, reject ) => {
+			const cwd = process.cwd();
+			const newCwd = path.join( cwd, data.mgit.packages, data.name );
```

Comment:
In order to make this tool more useful the cwd should not be taken from the `process.cwd()` but should be the directory which holds `mgit.json`. Just like git does with `.git/`. 

It doesn't make sense to write a code now which resolves the cwd to `mgit.json`'s directory, but at least keep this in mind.